ov/ov.go

```diff
@@ -20,19 +20,21 @@ package ov
 import (
 	"errors"
 	"fmt"
-
 	"github.com/HewlettPackard/oneview-golang/rest"
 	"github.com/docker/machine/libmachine/log"
 )
 
+var min_api_version int = 600
```

Comment:
Please remove this line as it doesn't have any usage

Reply:
Done